Sources/Xcodeproj/pbxproj().swift

```diff
@@ -56,10 +56,25 @@ public func pbxproj(srcroot: String, projectRoot: String, xcodeprojPath: String,
     print("            sourceTree = '<group>';")
     print("        };")
 
+////// Reference directories
+
+    var folderRefs = ""
+    for directoryReference in directoryReferences {
+        let folderRef = fileRef(inProjectRoot: directoryReference.basename, srcroot: srcroot)
+        folderRefs.append("\(folderRef.0),")
+        print("        \(folderRef.0) = {")
+        print("            isa = PBXFileReference;")
+        print("            lastKnownFileType = folder;")
+        print("            name = '\(folderRef.1)';")
+        print("            path = '\(Path(folderRef.2).relative(to: projectRoot))';")
+        print("            sourceTree = '<group>';")
+        print("        };")
+    }
+
 ////// root group
     print("        \(rootGroupReference) = {")
     print("            isa = PBXGroup;")
-    print("            children = (\(packageSwift.0), \(configsGroupReference), \(sourcesGroupReference), \(dependenciesGroupReference), \(testsGroupReference), \(productsGroupReference));")
+    print("            children = (\(packageSwift.0), \(configsGroupReference), \(sourcesGroupReference), \(folderRefs) \(dependenciesGroupReference), \(testsGroupReference), \(productsGroupReference));")
```

Comment:
This is an ordered list, I'm not sure if we want to put the "blue folder" references there but I personally like to have Dependencies and Products at the very bottom.